.github/scripts/build.sh

```diff
@@ -3,16 +3,16 @@
 source `dirname ${BASH_SOURCE[0]}`/config-mingw.sh
 
 .github/scripts/binutils/build.sh
-if [ $BUILD_MINGW ]; then
+if [ $BUILD_MINGW = 1 ]; then
```

Comment:
Changes in this file will be reverted once https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/90 will be merged.